<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>651</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="MainLayout">
    <property name="leftMargin">
     <number>25</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>25</number>
    </property>
    <property name="bottomMargin">
     <number>25</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="ItemViewLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="ItemTitleLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="TitleLabel1">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Item:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="EditItemToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>192</height>
           </size>
          </property>
          <property name="text">
           <string>{ICON}</string>
          </property>
          <property name="iconSize">
           <size>
            <width>192</width>
            <height>192</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="ItemNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>{NAME_N_COUNT}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="MainLayoutSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="RewardFrame">
      <layout class="QHBoxLayout" name="RewardLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="SettingNameLabel1">
         <property name="text">
          <string>Reward:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="RewardComboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="CraftingTimeFrame">
      <layout class="QHBoxLayout" name="CraftingTimeLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="SettingNameLabel2">
         <property name="text">
          <string>Crafting Time:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="CraftingTimeSpinBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="MainLayoutSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="IngredientsLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="IngredientsTitleLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="TitleLabel2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ingredients:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="EditIngredientToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>E</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="IngredientsTitleLayoutSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>4</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="RemoveIngredientToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>â€“</string>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="AddIngredientToolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="IngredientList">
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="BackDoneButtonsLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="BackButton">
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DoneButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Done</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>CraftingTimeSpinBox</tabstop>
  <tabstop>IngredientList</tabstop>
  <tabstop>EditIngredientToolButton</tabstop>
  <tabstop>AddIngredientToolButton</tabstop>
  <tabstop>RemoveIngredientToolButton</tabstop>
  <tabstop>DoneButton</tabstop>
  <tabstop>BackButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
